server/data/data-file/src/test/java/org/apache/james/user/file/UsersFileRepositoryTest.java

```diff
@@ -103,15 +105,15 @@ public File getFile(String fileURL) throws FileNotFoundException {
         return res;
     }
     
-    /* Disable testUpperCaseSameUser test.
+    /* Disable addUserShouldThrowWhenSameUsernameWithDiferentCase test.
```

Comment:
duplication => outdated content
Here you see your first occurrence of this problem : test name is already outdated !
Please remove this kind of comments.